src/org/ods/util/HelmStatusSimpleData.groovy

```diff
@@ -0,0 +1,186 @@
+package org.ods.util
+
+import com.cloudbees.groovy.cps.NonCPS
+import groovy.transform.TypeChecked
+
+@TypeChecked
+class HelmStatusSimpleData {
+
+    String name
+    String version
+    String namespace
+    String status
+    String description
+    String lastDeployed
+    /**
+     * Resources names by kind
+     */
+    Map<String, List<String> > resourcesByKind
+
+    @SuppressWarnings(['Instanceof'])
+    @NonCPS
+        static HelmStatusSimpleData fromJsonObject(Object object) {
+        try {
+            def rootObject = ensureMap(object, "")
```

Comment:
Why not declaring the parameter as Map data and removing the ensureMap check (and the method itself)?

Reply:
At least in one case, object parameter is the result of parsing JSON text via JsonSlurperClassic().parseText(helmStdout). This returns an Object by default. That object could be a List, a Map or some other object types, not necessarily a Map.